src/scripts/site.js

```diff
@@ -310,122 +310,234 @@ var AppInfo = {};
+            "peoplecardbuilder": "components/cardbuilder/peoplecardbuilder",
+            "chaptercardbuilder": "components/cardbuilder/chaptercardbuilder",
+            "deleteHelper": "components/deletehelper",
+            "tvguide": "components/guide/guide",
+            "guide-settings-dialog": "components/guide/guide-settings",
+            "loadingDialog": "components/loadingdialog/loadingdialog",
+            "slideshow": "components/slideshow/slideshow",
+            "objectassign": "components/polyfills/objectassign",
+            "focusPreventScroll": "components/polyfills/focusPreventScroll",
+            "userdataButtons": "components/userdatabuttons/userdatabuttons",
+            "emby-playstatebutton": "components/userdatabuttons/emby-playstatebutton",
+            "emby-ratingbutton": "components/userdatabuttons/emby-ratingbutton",
+            "listView": "components/listview/listview",
+            "indicators": "components/indicators/indicators",
+            "viewSettings": "components/viewsettings/viewsettings",
+            "filterMenu": "components/filtermenu/filtermenu",
+            "sortMenu": "components/sortmenu/sortmenu",
+            "idb": "components/idb",
+            "sanitizefilename": "components/sanitizefilename",
+            "toast": "components/toast/toast",
+            "scrollHelper": "components/scrollhelper",
+            "touchHelper": "components/touchhelper",
+            "appSettings": "scripts/settings/appSettings",
+            "userSettingsBuilder": "scripts/settings/userSettingsBuilder",
+            "userSettings": "scripts/settings/userSettings",
+            "imageUploader": "components/imageuploader/imageuploader",
+            "htmlMediaHelper": "components/htmlMediaHelper",
+            "viewContainer": "components/viewContainer",
+            "dialogHelper": "components/dialogHelper/dialogHelper",
+            "skinManager": "components/skinManager",
+            "scrollManager": "components/scrollManager",
+            "autoFocuser": "components/autoFocuser",
+            "appFooter": "components/appfooter/appfooter",
+            "playbackManager": "components/playback/playbackmanager",
+            "layoutManager": "components/layoutManager",
+            "lazyLoader": "components/lazyloader/lazyloader-intersectionobserver",
+            "castSenderApiLoader": "scripts/castSenderApiLoader",
+            "confirm": "components/confirm/confirm",
+            "prompt": "components/prompt/prompt",
+            "viewManager": "components/viewManager/viewManager",
+            "appRouter": "scripts/appRouter",
+            "connectionManager": "scripts/connectionManager",
+            "apiClientResolver": "scripts/apiClientResolver",
+            "serverNotifications": "components/serverNotifications",
+            "keyboardnavigation": "components/input/keyboardnavigation",
+            "mouseManager": "components//input/mouseManager"
```

Comment:
I suppose you mean ``"components/input/mouseManager"``

Reply:
Indeed !